packages/terra-responsive-element/CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ Unreleased
 ### Changed
 * Made required updates to consume terra-toolkit v5 and terra-dev-site v5
 
+### Fixed
+* Repititive Resizing when scrollbars appear between two breakpoints
```

Comment:
Repetitive

Reply:
We'll also want to move this changelog entry into the unreleased section